src/components/Dialog/Dialog.scss

```diff
@@ -7,113 +7,194 @@
 // --------------------------------------------------
 // Dialog styles
 
+$Dialog-lgHeader-backgroundColor: $ms-color-themePrimary;
+$Dialog-default-min-width: 288px;
+$Dialog-default-max-width: 340px;
+
+// Mixin for IE9 specific styles
+@mixin dialogPositioningIE9Fallback {
+  vertical-align: middle;
+  display: inline-block;
+}
+
 .ms-Dialog {
   @include ms-baseFont;
-  @include drop-shadow(0, 0, 5px, 0, .4);
+  background-color: transparent;
+  position: fixed;
+  height: 100%;
+  width:  100%;
+  top:  0;
+  @include left(0);
+
+  // Fallback for IE9
+  display: block;
+  font-size: 0;
```

Comment:
We should consider removing this, since it will hide elements that don't have a font-size explicitly set (child elements without a font-size will inherit this size). See https://github.com/OfficeDev/office-ui-fabric-js/issues/11